src/views/Schedule2.vue

```diff
@@ -32,9 +35,15 @@ export default Vue.extend({
   data() {
     return {
       ratio: 2.6,
-      width: 195
+      width: 195,
+      nowBarOffset: 0
     };
   },
+  created() {
+    const v = this as any;
+    setInterval(v.updateNowBarOffset, 5 * 60 * 1000);
```

Comment:
If we use `setInterval` we should also use `clearInterval`

Reply:
Fixed